specification/mobilenetwork/resource-manager/Microsoft.MobileNetwork/preview/2022-01-01-preview/packetCoreControlPlane.json

```diff
@@ -0,0 +1,396 @@
+        "description": "Lists all the packetCoreControlPlanes in a resource group.",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Request successful. The operation returns a list of packet core control plane resources.",
+            "schema": {
+              "$ref": "#/definitions/PacketCoreControlPlaneListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "List packet core control planes in resource group": {
+            "$ref": "./examples/PacketCoreControlPlaneListByResourceGroup.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    }
+  },
+  "definitions": {
+    "PacketCoreControlPlane": {
+      "type": "object",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/PacketCoreControlPlanePropertiesFormat",
+          "description": "Packet core control plane Properties."
+        },
+        "systemData": {
+          "x-ms-client-flatten": true,
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/systemData",
+          "readOnly": true,
+          "description": "Metadata pertaining to creation and last modification of the resource."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/TrackedResource"
+        }
+      ],
+      "required": [
+        "properties"
+      ],
+      "description": "Packet core control plane resource."
+    },
+    "PacketCoreControlPlanePropertiesFormat": {
+      "description": "PacketCoreControlPlane properties.",
+      "type": "object",
+      "properties": {
+        "provisioningState": {
+          "readOnly": true,
+          "$ref": "./common.json#/definitions/ProvisioningState",
+          "description": "The provisioning state of the packet core control plane resource."
+        },
+        "mobileNetwork": {
+          "$ref": "./common.json#/definitions/MobileNetworkResourceId",
+          "description": "Mobile network that this packet core control plane belongs to"
+        },
+        "customLocation": {
```

Comment:
why isn't this specified in the top level "extendedLocation" property?  https://armwiki.azurewebsites.net/api_contracts/extendedLocations.html#resource-api

Reply:
Our resource provider does not support extended locations directly. The customLocation property in the API is passed down to another resource provider (Microsoft.HybridNetwork) that does support extended locations. Is that acceptable?

Reply:
Probably.  But please verify with HybridNetwork and/or the extendedLocation folks that access can't be violated here to extendedLocations that the customer shouldn't normally have access to